ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master__cnv2.5_4.6.yaml

```diff
@@ -0,0 +1,156 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+canonical_go_repository: kubevirt.io/kubevirt
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- artifact_dir: /tmp/artifacts
+  as: e2e
+  cron: 2 3 * * *
+  steps:
+    cluster_profile: azure4
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        set -ex
+
+        # disable default catalog source
+        oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
+
+        authfile=/tmp/authfile
+        # get authfile from cluster
+        oc get secret/pull-secret -n openshift-config -o json | jq -r '.data.".dockerconfigjson"' | base64 -d > "$authfile"
+
+        # login into brew image registry
+        docker login --username "$BREW_IMAGE_REGISTRY_USERNAME" --password-stdin brew.registry.redhat.io < $BREW_IMAGE_REGISTRY_TOKEN_PATH
+
+        # update authfile with credentials from docker config
+        jq -c '.auths + '"$(jq '{"brew.registry.redhat.io": .auths."brew.registry.redhat.io" }' $HOME/.docker/config.json)"' | {"auths": .}' "$authfile" > "${authfile}.new"
+
+        # update cluster secret
+        oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson="${authfile}.new"
+
+        # install imageContentSourcePolicy
+        oc create -f - << EOF
+        apiVersion: operator.openshift.io/v1alpha1
+        kind: ImageContentSourcePolicy
+        metadata:
+          name: brew-registry
+        spec:
+          repositoryDigestMirrors:
+          - mirrors:
+            - brew.registry.redhat.io
+            source: registry.redhat.io
+          - mirrors:
+            - brew.registry.redhat.io
+            source: registry.stage.redhat.io
+          - mirrors:
+            - brew.registry.redhat.io
+            source: registry-proxy.engineering.redhat.com
+        EOF
+
+        # wait for machineconfigpool to get updated
+        oc wait mcp/master --for condition=updated --timeout=600s
+        oc wait mcp/worker --for condition=updated --timeout=600s
+
+        # install index image
+        oc create -f - << EOF
+        apiVersion: operators.coreos.com/v1alpha1
+        kind: CatalogSource
+        metadata:
+          name: brew-catalog-source
+          namespace: openshift-marketplace
+        spec:
+          sourceType: grpc
+          image: brew.registry.redhat.io/rh-osbs/iib-pub-pending:v$CNV_VERSION
```

Comment:
In this case you are referring to the general staging index image, thus `${CNV_VERSION}` should be 4.6 in case of CNV 2.5 or 4.7 in case of CNV 2.6

Reply:
additional info:
https://docs.engineering.redhat.com/display/CFC/Test